conf/.htaccess.security

```diff
@@ -1,4 +1,6 @@
 # BEGIN core-standards:security
+RedirectMatch 403 /wp-includes/.+\.php$
```

Comment:
To my knowledge, we do not have a routine for updating these snippets yet, so just updating the template will not update the site. We added the begin/end delimiters to enable that. Could you look into that?